os-lifecycle-policy.md

```diff
@@ -17,6 +16,7 @@ When an operating system version goes out of standard support, we stop testing t
 
 The following support documents list historical operating system support for out-of-support .NET Core versions:
 
+* [.NET Core 2.2 supported OS versions](release-notes/2.2/2.2-supported-os.md)
```

Comment:
This list is reverse order and the one in microsoft-support.md is chronological order. We should pick one approach. I have a preference for reverse order.

Reply:
Fixed the order in microsoft-support.md